app/src/main/java/org/zalando/nakadi/config/AuthenticationConfig.java

```diff
@@ -154,7 +155,7 @@ public Map<String, Object> getMap(final String accessToken) {
                     .header(HttpHeaders.AUTHORIZATION, String.format(BEARER_TOKEN_TEMPLATE, accessToken)).build();
             try {
                 return restTemplate.exchange(entity, TOKENINFO_MAP).getBody();
-            } catch (HttpStatusCodeException e) {
+            } catch (HttpStatusCodeException | ResourceAccessException e ) {
```

Comment:
from what I can see, `ResourceAccessException` is thrown when `RestTemplate` encounters a IOException

Reply:
is there any test to see that it works as expected? current test does not represent that.